crates/rv/src/config.rs

```diff
@@ -56,15 +56,15 @@ impl Config {
     }
 
     pub fn current_ruby(&self) -> Option<Ruby> {
-        if let Ok(request) = self.ruby_request() {
+        if let Ok(request) = self.default_version() {
             self.matching_ruby(&request)
         } else {
             None
         }
     }
 
-    pub fn ruby_request(&self) -> Result<RubyRequest> {
```

Comment:
hm, I think I'm coming around to `ruby_request` after all since it matches the underlying RubyRequest return type.